Gateway/Http/Client/TransactionPayment.php

```diff
@@ -26,16 +26,15 @@
 use Adyen\Service\Checkout\PaymentsApi;
 use Magento\Framework\Exception\AlreadyExistsException;
 use Magento\Framework\Exception\NoSuchEntityException;
-use Magento\Payment\Gateway\Http\ClientInterface;
 use Magento\Payment\Gateway\Http\TransferInterface;
 use Magento\Store\Model\StoreManagerInterface;
 
-class TransactionPayment implements ClientInterface
+class TransactionPayment extends BaseTransaction
```

Comment:
Abstract `requestHeaders()` method has not been called in this class. Do we still need to extend `BaseTransaction`?